recipes/pyvisfile/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pyvisfile" %}
+{% set version = "2016.1" %}
+{% set sha256 = "1b1f4eec567b2a16893e61c7b36f2ecf63823dad12a802ef5427a243e44795fb" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - pytools
+        - six
```

Comment:
Please cut anything not needed at build time from the build dependencies (probably just `python` and `setuptools`).